setup.py

```diff
@@ -464,13 +451,39 @@ def getfn(fn, prefix = "src/umich"):
               )
 ]
 
-setup(name = NAME,
-    version = VERSION,
-    description = "Gao Wang's statgen library",
-    author = "Gao Wang",
-    packages = [NAME, NAME + ".egglib"],
-    package_dir = {NAME:'src'},
-    package_data = {NAME + ".egglib":['apps.conf']},
-    cmdclass = {'build_py': build_py},
-    ext_modules = CSTATGEN_MODULE + EGGLIB_MODULE + NUM_MODULE + ASSOTESTS_MODULE
-)
+if sys.version_info.major == 2:
```

Comment:
can you improve the logic such that you have just one `setup()` call? eg you can make `EGGLIB_MODULE` empty `[]` when it is not python 2?

Reply:
Have updated.

Reply:
The patch looks reasonable. Let's fix the libx.so installation issue before merging it.